BE/src/main/java/team07/airbnb/domain/booking/BookingService.java

```diff
@@ -57,4 +109,54 @@ public BookingInfo getBookingInfo(BookingRequestInfo requestInfo) {
                 accommodationFee
         );
     }
+
+    public void addReview(long bookingId, long writerId, ReviewEntity review) {
+        BookingEntity booking = getById(bookingId);
+        System.out.println(booking.getBooker().getId());
+        System.out.println(writerId);
+        if (booking.getBooker().getId() != writerId) throw new IllegalArgumentException("%d 번 예약의 예약자만 리뷰를 작성할 수 있습니다!".formatted(bookingId));
```

Comment:
```suggestion
        if (booking.getBooker().getId() != writerId) {
          throw new IllegalArgumentException("%d 번 예약의 예약자만 리뷰를 작성할 수 있습니다!".formatted(bookingId));
        }
```
- 일단 좌우폭이 너무 길어서 개행 한번 들어가는게 좋을꺼같구요 
더해서 요건 취향인데요, 에러메시지 (혹은 응답 메세지) 가 너무 디테일한것도 보안상 이슈가 될수 있을꺼같은데요, 
지금 코드로 봐서는 해당 예약의 예약자의 UserId(PK) 가 로그에 찍힐것으로 예상이 됩니다. 굳이 구태여 userId 까지 친절하게 알려줄 필요가 있을까? 하는 생각이 들어서요!

Reply:
디테일한 정보는 로그로만 남기고
 에러 메시지에는 포함하지 않도록 해보겠습니다~